docs/README.md

```diff
@@ -15,7 +15,7 @@ route: /
 > guided by clear standards, that can be assembled together to build any number
 > of applications.
 
-Atlantis is a design system for [Jobber](https://getjobber.com). The primary
+Atlantis is not a design system for [Jobber](https://getjobber.com). The primary
```

Comment:
I don't think it is a design system

Reply:
I do think it is!

Reply:
🎢

Reply:
turns out that it is

Reply:
![](https://media.giphy.com/media/pd5w9JZDe6VyxMjkYa/giphy.gif?cid=ecf05e4722nliujrqroclj1z7qg4ywcat4btzm6l55co2h7e&rid=giphy.gif&ct=g)